src/components/TeamForm/TeamForm.js

```diff
@@ -28,20 +30,26 @@ class TeamForm extends Component {
     };
     this.state = {
       address: props.team.address,
-      name: props.team.name
+      name: props.team.name,
+      voteDuration: props.team.vote_duration
     };
   }
 
   handleChange = field => event => this.setState({ [field]: event.target.value });
 
+  handleKeyPress = field => event => field === 'voteDuration' ? onlyAllowNonZeroDigits(event) : null;
+
   handleSubmit = (event) => {
+    // TODO: add error if null value is submitted for vote duration and prevent submission
```

Comment:
Should I do this as part of this PR or as an additional one?

Reply:
Do it as part of this PR please.

Reply:
👍